installer/templates/phx_web/router.ex

```diff
@@ -28,21 +37,15 @@ defmodule <%= web_namespace %>.Router do
   # If your application does not have an admins-only section yet,
   # you can use Plug.BasicAuth to set up some basic authentication
   # as long as you are also using SSL (which you should anyway).
-  if Mix.env() == :dev do
-    import Phoenix.LiveDashboard.Router
+  if Mix.env() in [:dev, :test] do
+    import Phoenix.LiveDashboard.Router<%= unless html && live do %>
+
+    defdelegate fetch_live_flash(conn, opts), to: Phoenix.LiveView.Router<% end %>
```

Comment:
We don't use flash on the dashboard, so we don't need to add this. :)

Reply:
There's a `put_flash` in there for a node disconnection message, or at least there was yesterday :)

Reply:
@mcrumm good call! Note we don't read from the flash anywhere (no calls to @flash or live_flash), so it is not used anyway. So I think we can remove this here and we find an alternative for the flash message on the dashboard. :)